v4l-utils is broken on unstable

**Describe the bug**
```
error: --- Error --- build-remote
build of '/nix/store/x4nz3kqn24vlz257hm8mnsvrn6ggfsrs-v4l-utils-1.20.0.drv' on 'ssh://root@nixos' failed: builder for '/nix/store/x4nz3kqn24vlz257hm8mnsvrn6ggfsrs-v4l-utils-1.20.0.drv' failed with exit code 2
error: --- Error --------------------------------------------------------------------------------------------------------------------------------------- nix
builder for '/nix/store/x4nz3kqn24vlz257hm8mnsvrn6ggfsrs-v4l-utils-1.20.0.drv' failed with exit code 1; last 10 log lines:
  ../../utils/libcecutil/cec-gen.pl ../../include/linux/cec.h ../../utils/libcecutil/cec-htng.h ../../include/linux/cec-funcs.h ../../utils/libcecutil/cec-htng-funcs.h
  /nix/store/j8vysakw78bpgngba32hfwwikqda9yx2-bash-4.4-p23/bin/bash: ../../utils/libcecutil/cec-gen.pl: /usr/bin/perl: bad interpreter: No such file or directory
  make[3]: *** [Makefile:782: gen.intermediate] Error 126
  make[3]: Leaving directory '/build/v4l-utils-1.20.0/utils/libcecutil'
  make[2]: *** [Makefile:482: all-recursive] Error 1
  make[2]: Leaving directory '/build/v4l-utils-1.20.0/utils'
  make[1]: *** [Makefile:591: all-recursive] Error 1
  make[1]: Leaving directory '/build/v4l-utils-1.20.0'
  make: *** [Makefile:518: all] Error 2
  builder for '/nix/store/x4nz3kqn24vlz257hm8mnsvrn6ggfsrs-v4l-utils-1.20.0.drv' failed with exit code 2
error: --- Error --------------------------------------------------------------------------------------------------------------------------------------- nix
1 dependencies of derivation '/nix/store/2544nk822aa38adwvdgk8kc1zyxf1wgd-system-path.drv' failed to build

```

`  /nix/store/j8vysakw78bpgngba32hfwwikqda9yx2-bash-4.4-p23/bin/bash: ../../utils/libcecutil/cec-gen.pl: /usr/bin/perl: bad interpreter: No such file or directory`

we need another `patchShebangs`. PR incoming...

I'm on 8e2b14aceb1d40c7e8b84c03a7c78955359872bb

--- a/pkgs/os-specific/linux/v4l-utils/default.nix
+++ b/pkgs/os-specific/linux/v4l-utils/default.nix
@@ -41,6 +41,7 @@ in stdenv.mkDerivation rec {
 
   postPatch = ''
     patchShebangs utils/cec-ctl/msg2ctl.pl
+    patchShebangs utils/libcecutil/cec-gen.pl
   '';
 
   meta = with stdenv.lib; {

